config/manager/kustomization.yaml

```diff
@@ -11,6 +11,6 @@ configMapGenerator:
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
-- name: controller
-  newName: sbo
-  newTag: latest
+- digest: sha256:c46fc750d0234f1624607e321053f7af94034e649e820a5d4e784a1117787d70
+  name: controller
+  newName: 192.168.49.2:5000/sbo
```

Comment:
@pedjak I believe this is a accidentally part of the PR. The `newName` is likely for you local minikube and the digest will also change with every change...